nx-X11/programs/Xserver/hw/nxagent/Events.c

```diff
@@ -4316,16 +4299,16 @@ int nxagentClipAndSendExpose(WindowPtr pWin, void * ptr)
   {
     exposeRgn = RegionCreate(NULL, 1);
 
+    #ifdef DEBUG
     box = *RegionExtents(remoteExposeRgn);
 
-    #ifdef DEBUG
     fprintf(stderr, "nxagentClipAndSendExpose: Root expose extents: [%d] [%d] [%d] [%d].\n",
                 box.x1, box.y1, box.x2, box.y2);
     #endif
```

Comment:
Squash this into one `#ifdef DEBUG` block, while we're at it anyway. No need to end and begin a new block right away.